packages/carbon-web-components/src/components/notification/notification.mdx

```diff
@@ -27,8 +30,8 @@ import '@carbon/web-components/es/components/notification/index.js';
 @use '@carbon/styles/scss/components/notification';
 ```
 
-<Description markdown={`${cdnJs({ components: ['notification'] })}`} />
-<Description markdown={`${cdnCss()}`} />
+<Markdown>{`${cdnJs({ components: ['notification'] })}`}</Markdown>
```

Comment:
Made this change after having issues with `<Meta />` before realizing in this case I didn't need it. Also I realized that the `markdown` attribute was deprecated. More details on the migration path I followed here: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#meta-block